### ISSUE TITLE
Toggling Header Styles

### ISSUE THREAD

**Issue by KAUSHIKTECHIE** [Andrew-Chen-Wang/RichEditorView#11]
How can we make the headers to toggle like the bold,italics meaning how to remove heading when clicked for second time.
The header/size buttons should also be a toggle; doesn’t allow to toggle back to default size.

**Comment by markosrx** [comment]
+1

**Comment by Andrew-Chen-Wang** [comment]
Hopefully this helps!:
```js
RE.setHeading = function(heading) {
    var sel = document.getSelection();
    var el = sel.getRangeAt(0).startContainer;
    document.execCommand('formatBlock', false, el.parentNode.tagName === `H${heading}` ? '<p>' : '<h' + heading + '>');
};
```

Note to self: document.execCommand is deprecated, but there's literally no alternative... we'll have to see what happens after finals.

<details><summary>Other notes for future me</summary>
The sister repo also doesn't seem to have a solution unfortunately... https://github.com/wasabeef/richeditor-android/blob/ce152b910a59c66863baa68b007ac67e6b9de671/richeditor/src/main/assets/rich_editor.js#L155-L157
</details>

**Comment by Andrew-Chen-Wang** [comment]
Any ideas where else I can add this besides the heading?

**Comment by Andrew-Chen-Wang** [comment]
Fixed in ed734f6f9bb2bb2e09efe183764b488d50d3bcf1
